test/execute.js

```diff
@@ -454,14 +454,143 @@ describe('execute', () => {
       // Then
       expect(req).toEqual({
         url: 'http://swagger.io/one',
-        headers: {
-          'content-type': 'application/josh',
-        },
+        headers: {},
         credentials: 'same-origin',
         method: 'GET'
       })
     })
 
+    it('should not add content-type with no form-data or body param', function () {
```

Comment:
I would like to see a test, where _no_ contentType is added. As one of our current issues, is that GET requests get a `content-type: application/json`. And I'd like to make sure that doesn't happen in swagger-js ( I'm pretty sure it doesn't, but still ).